javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,18 @@
+var fullName = 'Youssef Najjarine';
+var isCool = false;
+var totalPets = 2;
+var unknown = null;
+var never =
```

Comment:
This works, but I would remove the `=` here as you don't want to use one without an assignment operation